kaggle_environments/envs/chess/chess.py

```diff
@@ -0,0 +1,108 @@
+import chess
+import random
+import json
+from os import path
+
+def random_agent(obs):
+  """
+  Selects a random legal move from the board.
```

Comment:
Optional but docstrings and/or type hints are helpful-- consider adding them to the other functions.